spring-cloud-sleuth-instrumentation/src/test/java/org/springframework/cloud/sleuth/instrument/web/client/feign/FeignContextBeanPostProcessorTest.java

```diff
@@ -0,0 +1,21 @@
+package org.springframework.cloud.sleuth.instrument.web.client.feign;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.cloud.openfeign.FeignContext;
+import org.springframework.context.ApplicationContext;
+
+
+class FeignContextBeanPostProcessorTest {
```

Comment:
If you want to you can leverage `MockitoExtensions` and inject `@Mock` into fields. There are other examples out there in the repo.